commons/ava_testsuite/rpc_manager.go

```diff
@@ -0,0 +1,147 @@
+package ava_testsuite
+
+import (
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_default_testnet"
+	"github.com/kurtosis-tech/ava-e2e-tests/gecko_client"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+const (
+	GENESIS_USERNAME = "genesis"
+	GENESIS_PASSWORD = "genesis34!23"
+	ACCEPTED_STATUS = "Accepted"
+)
+
+type RpcManager struct {
+	client *gecko_client.GeckoClient
+	testNet *ava_default_testnet.TestNet
```

Comment:
I think we have two ways forward here:
1) don't pass in the TestNet as a parameter, since there's only one TestNet and it's a constant (`DefaultTestNet`) so just use that at all points where you'd normally have used `rpcManager.testNet` (my preferred way)
2) Emphasize that the `TestNet` type is generic by moving it to its own file (`ava_test_net` maybe) and that `ava_default_testnet` is but one implementation. I prefer this less because, to my understanding, we're always going to roll with DefaultTestNet

Reply:
I'm going to move forward with #2 and do that refactor

Reply:
did the first option